src/button-group.test.miscellaneous.ts

```diff
@@ -0,0 +1,263 @@
+import { expect, test } from './playwright/test.js';
+
+test('defines itself', { tag: '@miscellaneous' }, async ({ mount, page }) => {
+  await mount(
+    () =>
+      html`<glide-core-button-group label="Label">
+        <glide-core-button-group-button
+          label="Label"
+        ></glide-core-button-group-button>
+      </glide-core-button-group>`,
+  );
+
+  const host = page.locator('glide-core-button-group');
+
+  await expect(host).toBeInTheCustomElementRegistry('glide-core-button-group');
+});
+
+test(
+  'passes certain properties to its buttons when they are set initially',
+  { tag: '@miscellaneous' },
+  async ({ mount, page }) => {
+    await mount(
+      () =>
+        html`<glide-core-button-group
+          label="Label"
+          orientation="vertical"
+          variant="icon-only"
+        >
+          <glide-core-button-group-button label="One">
+            <div slot="icon">★</div>
+          </glide-core-button-group-button>
+
+          <glide-core-button-group-button label="Two">
+            <div slot="icon">⚙</div>
+          </glide-core-button-group-button>
+        </glide-core-button-group>`,
+    );
+
+    const buttons = page.locator('glide-core-button-group-button');
+
+    await expect(buttons.nth(0)).toHaveJSProperty(
+      'privateOrientation',
+      'vertical',
+    );
+
+    await expect(buttons.nth(0)).toHaveJSProperty(
+      'privateVariant',
+      'icon-only',
+    );
+
+    await expect(buttons.nth(1)).toHaveJSProperty(
+      'privateOrientation',
+      'vertical',
+    );
+
+    await expect(buttons.nth(1)).toHaveJSProperty(
+      'privateVariant',
+      'icon-only',
+    );
+  },
+);
+
+test(
+  'passes certain properties to its buttons when they are set programmatically',
+  { tag: '@miscellaneous' },
+  async ({ mount, page, setProperty }) => {
+    await mount(
+      () =>
+        html`<glide-core-button-group label="Label">
+          <glide-core-button-group-button label="One">
+            <div slot="icon">★</div>
+          </glide-core-button-group-button>
+
+          <glide-core-button-group-button label="Two">
+            <div slot="icon">⚙</div>
+          </glide-core-button-group-button>
+        </glide-core-button-group>`,
+    );
+
+    const host = page.locator('glide-core-button-group');
+    const buttons = page.locator('glide-core-button-group-button');
+
+    await setProperty(host, 'orientation', 'vertical');
+    await setProperty(host, 'variant', 'icon-only');
+
+    await expect(buttons.nth(0)).toHaveJSProperty(
+      'privateOrientation',
+      'vertical',
+    );
+
+    await expect(buttons.nth(0)).toHaveJSProperty(
+      'privateVariant',
+      'icon-only',
+    );
+
+    await expect(buttons.nth(1)).toHaveJSProperty(
+      'privateOrientation',
+      'vertical',
+    );
+
+    await expect(buttons.nth(1)).toHaveJSProperty(
+      'privateVariant',
+      'icon-only',
+    );
+  },
+);
+
+test(
+  'selects initially the first enabled button',
```

Comment:
Maybe just me, but this reads a little oddly. Any better?


```suggestion
  'initially selects the first enabled button',
```